src/component/Table.astro

```diff
@@ -21,11 +21,11 @@ const { downloadData } = Astro.props;
           <td class="pl-2 pt-3">
             <Button text={item.buttonText} type={item.type ? item.type : 'primary'} url={item.downloadUrl} />
           </td> 
-          <td class="pl-4">{item.size}</td>   
           <td class="pl-4">
             {item.description} {item.descriptionBold ? 
               ( <text><b>{item.descriptionBold}</b>{item.extraDescription}</text> ) : null} 
           </td>
+          <td class="pl-4"><p>{item.size}</p><p class="hash">{item.hashA}</p><p class="hash">{item.hashB}</p></td>   
```

Comment:
Dumb q, what does it do for items without hashA or hashB? (Whitespace fine with me, error message less so...)

Reply:
The pages without hashes look like this:
![image](https://github.com/AcademySoftwareFoundation/dpel-website/assets/169599/813b78e3-3f48-416f-befc-640527988da6)